L1Trigger/Phase2L1ParticleFlow/python/l1ctLayer1_cff.py

```diff
@@ -2,10 +2,10 @@
 
 import math
 
-from L1Trigger.Phase2L1ParticleFlow.pfTracksFromL1Tracks_cfi import pfTracksFromL1Tracks
-from L1Trigger.Phase2L1ParticleFlow.pfClustersFromL1EGClusters_cfi import pfClustersFromL1EGClusters
-from L1Trigger.Phase2L1ParticleFlow.pfClustersFromCombinedCalo_cff import pfClustersFromCombinedCaloHCal, pfClustersFromCombinedCaloHF
-from L1Trigger.Phase2L1ParticleFlow.pfClustersFromHGC3DClusters_cfi import pfClustersFromHGC3DClusters
+from L1Trigger.Phase2L1ParticleFlow.pfTracksFromL1Tracks_cfi import l1tPFTracksFromL1Tracks
+from L1Trigger.Phase2L1ParticleFlow.pfClustersFromL1EGClusters_cfi import l1tPFClustersFromL1EGClusters
+from L1Trigger.Phase2L1ParticleFlow.pfClustersFromCombinedCalo_cff import l1tPFClustersFromCombinedCaloHCal, l1tPFClustersFromCombinedCaloHF
+from L1Trigger.Phase2L1ParticleFlow.pfClustersFromHGC3DClusters_cfi import l1tPFClustersFromHGC3DClusters
 
 from L1Trigger.Phase2L1ParticleFlow.l1TkEgAlgoEmulator_cfi import tkEgAlgoParameters,tkEgSorterParameters
```

Comment:
The objects in `l1TkEgAlgoEmulator_cfi` don't follow the naming convention?

Reply:
These are parameters; I have renamed modules, analyzers, and sequences.